app/schema.graphql

```diff
@@ -179,6 +179,15 @@ type DimensionWithValue {
   value: String
 }
 
+type DocumentEvaluation implements Evaluation {
+  name: String!
+  score: Float
+  label: String
+  explanation: String
+  spanId: String!
+  documentPosition: Int!
```

Comment:
docs: Adding descriptions for the fields would be useful

Reply:
will do